src/styles/main.scss

```
@@ -1,3 +1,74 @@
-body {
+$day-cell-size: 100px;
+$day-margin: 0.5px;
+$day-bckgrd-color: #eee;
+$day-border-width: 1px;
+$days-of-week: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7;
+$calendar-columns: 7;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width: ($day-cell-size * $calendar-columns)
+  + ($calendar-gap * $calendar-columns)
+  + ($calendar-padding * 2);
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__item {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: $calendar-width;
+    min-width: $calendar-width;
+    padding: $calendar-padding;
+
+    @each $day, $i in $days-of-week {
+      &--start-day-#{$day}>:first-child {
+        margin-left: calc((#{$i} - 1) * (#{$day-cell-size} + #{$calendar-gap}));
+      }
+    }
+
+    @for $day from 28 through 31 {
+      &--month-length-#{$day}>:nth-child(n + #{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    position: relative;
+    width: $day-cell-size;
+    border: $day-border-width solid black;
+    background-color: $day-bckgrd-color;
+    transition: all 0.5s;
```

specify what rules you'd like to change
```suggestion
    transition: background-color 0.5s, transform 0.5s;
```